pom.xml

```diff
@@ -16,7 +16,7 @@
   
   <properties>
     <java.version>1.8</java.version>
-    <adam.version>0.23.0</adam.version>
+    <adam.version>0.24.0</adam.version>
```

Comment:
Note this and other dependency versions are quite far behind.  With @fnothaft not very involved in ADAM development any more, we might want to discuss what it would take to get up to date.

Reply:
Got it. I am hoping that we can get this merge in now (in anticipation of upcoming paper submissions) and then try to update the ADAM dependency.

Reply:
Agreed, keep me in the loop about the paper deadlines (email username at berkeley.edu) and I'll try to help bring the dependencies up to date.